### ISSUE TITLE
Download of Java VM failed

### ISSUE THREAD

**Issue by barthel** [karakun/OpenWebStart#104]
OpenWebStart 1.0.0
Windows 10
no Java VM installed

```
[xxxxxxxxxx][ITW-APPLET][WARNING_ALL][Wed Nov 27 09:28:03 CET 2019][net.sourceforge.jnlp.util.logging.FileLog.getHeadlineHeader(FileLog.java:59)] NETX Thread# 7d00ca8, name Output controller consumer daemon writer-based impl.
[xxxxxxxxxx][ITW-APPLET][MESSAGE_ALL][Wed Nov 27 09:28:03 CET 2019][com.openwebstart.launcher.OpenWebStartLauncher] NETX Thread# 6bf2d08e, name main ITW Boot called with custom OwsJvmLauncher and args [-verbose, -jnlp, C:\Users\xxxxxxxxxx\AppData\Local\Microsoft\Windows\INetCache\IE\CUF5XDW5\my-application.DEV.jnlp].
[xxxxxxxxxx][ITW-APPLET][MESSAGE_ALL][Wed Nov 27 09:28:03 CET 2019][com.openwebstart.launcher.PhaseTwoWebStartLauncher] NETX Thread# 6bf2d08e, name main Starting OpenWebStart 1.0.0
[xxxxxxxxxx][ITW-APPLET][MESSAGE_ALL][Wed Nov 27 09:28:03 CET 2019][com.openwebstart.launcher.PhaseTwoWebStartLauncher] NETX Thread# 6bf2d08e, name main OpenWebStartLauncher called with args [-verbose, -jnlp, C:\Users\xxxxxxxxxx\AppData\Local\Microsoft\Windows\INetCache\IE\CUF5XDW5\my-application.DEV.jnlp].
[xxxxxxxxxx][ITW-APPLET][MESSAGE_DEBUG][Wed Nov 27 09:28:03 CET 2019][com.openwebstart.launcher.PhaseTwoWebStartLauncher] NETX Thread# 6bf2d08e, name main OS detected: Win[true], MacOS[false], Linux[false]
[xxxxxxxxxx][ITW-APPLET][MESSAGE_ALL][Wed Nov 27 09:28:03 CET 2019][net.sourceforge.jnlp.config.DeploymentConfiguration] NETX Thread# 6bf2d08e, name main Ico provider registered correctly.
[xxxxxxxxxx][ITW-APPLET][MESSAGE_ALL][Wed Nov 27 09:28:03 CET 2019][net.sourceforge.jnlp.config.DeploymentConfiguration] NETX Thread# 6bf2d08e, name main Ico provider registered correctly.
[xxxxxxxxxx][ITW-APPLET][MESSAGE_ALL][Wed Nov 27 09:28:03 CET 2019][net.sourceforge.jnlp.config.DeploymentConfiguration] NETX Thread# 6bf2d08e, name main Loading USER level properties from: file:/C:/Users/xxxxxxxxxx/.config/icedtea-web/deployment.properties
[xxxxxxxxxx][ITW-APPLET][MESSAGE_ALL][Wed Nov 27 09:28:03 CET 2019][net.sourceforge.jnlp.config.DeploymentConfiguration] NETX Thread# 6bf2d08e, name main Property 'ows.install4j.propertyUpdate' is unknown.
[xxxxxxxxxx][ITW-APPLET][MESSAGE_DEBUG][Wed Nov 27 09:28:03 CET 2019][net.sourceforge.jnlp.util.logging.FileLog] NETX Thread# 7d00ca8, name Output controller consumer daemon Attempting to log into: C:\Users\xxxxxxxxxx\.config\icedtea-web\log\itw-javantx-2019-11-27_09_28_03.601.log
[xxxxxxxxxx][ITW-APPLET][MESSAGE_ALL][Wed Nov 27 09:28:03 CET 2019][net.sourceforge.jnlp.config.DeploymentConfiguration] NETX Thread# 6bf2d08e, name main Loading USER level properties from: file:/C:/Users/xxxxxxxxxx/.config/icedtea-web/deployment.properties
[xxxxxxxxxx][ITW-APPLET][MESSAGE_ALL][Wed Nov 27 09:28:03 CET 2019][net.sourceforge.jnlp.config.DeploymentConfiguration] NETX Thread# 6bf2d08e, name main Property 'ows.install4j.propertyUpdate' is unknown.
[xxxxxxxxxx][ITW-APPLET][MESSAGE_DEBUG][Wed Nov 27 09:28:03 CET 2019][com.openwebstart.launcher.InitialConfigurationCheck] NETX Thread# 6bf2d08e, name main Checking if installation time (1574842294848) is after last initial config time (1574843032877)
[xxxxxxxxxx][ITW-APPLET][MESSAGE_DEBUG][Wed Nov 27 09:28:03 CET 2019][com.openwebstart.launcher.PhaseTwoWebStartLauncher] NETX Thread# 6bf2d08e, name main RelevantJavawsArgs: '[-verbose, -jnlp, C:\Users\xxxxxxxxxx\AppData\Local\Microsoft\Windows\INetCache\IE\CUF5XDW5\my-application.DEV.jnlp]'
[xxxxxxxxxx][ITW-APPLET][MESSAGE_DEBUG][Wed Nov 27 09:28:03 CET 2019][com.openwebstart.jvm.LocalRuntimeManager] NETX Thread# 6bf2d08e, name main Loading runtime cache from filesystem
[xxxxxxxxxx][ITW-APPLET][MESSAGE_DEBUG][Wed Nov 27 09:28:03 CET 2019][com.openwebstart.jvm.LocalRuntimeManager] NETX Thread# 6bf2d08e, name main Clearing runtime cache
[xxxxxxxxxx][ITW-APPLET][MESSAGE_DEBUG][Wed Nov 27 09:28:03 CET 2019][com.openwebstart.jvm.LocalRuntimeManager] NETX Thread# 6bf2d08e, name main Saving runtime cache to filesystem
[xxxxxxxxxx][ITW-APPLET][MESSAGE_ALL][Wed Nov 27 09:28:03 CET 2019][com.openwebstart.launcher.PhaseTwoWebStartLauncher] NETX Thread# 6bf2d08e, name main ITW Boot called with custom OwsJvmLauncher and args [-verbose, -jnlp, C:\Users\xxxxxxxxxx\AppData\Local\Microsoft\Windows\INetCache\IE\CUF5XDW5\my-application.DEV.jnlp].
[xxxxxxxxxx][ITW-APPLET][MESSAGE_ALL][Wed Nov 27 09:28:03 CET 2019][net.adoptopenjdk.icedteaweb.launch.JvmLauncherHolder] NETX Thread# 6bf2d08e, name main JvmLauncher set to com.openwebstart.launcher.OwsJvmLauncher.
[xxxxxxxxxx][ITW-APPLET][MESSAGE_DEBUG][Wed Nov 27 09:28:03 CET 2019][net.sourceforge.jnlp.runtime.JNLPRuntime] NETX Thread# 6bf2d08e, name main Forcing of offline set to: false
[xxxxxxxxxx][ITW-JAVAWS][MESSAGE_DEBUG][Wed Nov 27 09:28:04 CET 2019][net.adoptopenjdk.icedteaweb.client.parts.dialogs.security.SecurityDialogMessageHandler] NETX Thread# 5e4725bb, name NetxSecurityThread Starting security dialog thread
[xxxxxxxxxx][ITW-JAVAWS][ERROR_ALL][Wed Nov 27 09:28:04 CET 2019][net.sourceforge.jnlp.browser.BrowserAwareProxySelector] NETX Thread# 6bf2d08e, name main
Unable to use Firefox''s proxy settings. Using "DIRECT" as proxy type.
java.io.FileNotFoundException: C:\Users\xxxxxxxxxx\AppData\Roaming\Mozilla\Firefox\profiles.ini
	at net.sourceforge.jnlp.browser.FirefoxPreferencesFinder.find(FirefoxPreferencesFinder.java:88)
	at net.sourceforge.jnlp.browser.BrowserAwareProxySelector.parseBrowserPreferences(BrowserAwareProxySelector.java:151)
	at net.sourceforge.jnlp.browser.BrowserAwareProxySelector.initFromBrowserConfig(BrowserAwareProxySelector.java:114)
	at net.sourceforge.jnlp.browser.BrowserAwareProxySelector.initialize(BrowserAwareProxySelector.java:102)
	at net.sourceforge.jnlp.runtime.JNLPRuntime.initialize(JNLPRuntime.java:298)
	at net.sourceforge.jnlp.runtime.Boot.init(Boot.java:372)
	at net.sourceforge.jnlp.runtime.Boot.run(Boot.java:289)
	at net.sourceforge.jnlp.runtime.Boot.run(Boot.java:78)
	at java.security.AccessController.doPrivileged(Native Method)
	at net.sourceforge.jnlp.runtime.Boot.main(Boot.java:233)
	at com.openwebstart.launcher.PhaseTwoWebStartLauncher.main(PhaseTwoWebStartLauncher.java:81)
	at com.openwebstart.launcher.OpenWebStartLauncher.main(OpenWebStartLauncher.java:29)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.exe4j.runtime.LauncherEngine.launch(LauncherEngine.java:85)
	at com.exe4j.runtime.WinLauncher.main(WinLauncher.java:94)
	at com.install4j.runtime.launcher.WinLauncher.main(WinLauncher.java:25)
[xxxxxxxxxx][ITW-JAVAWS][MESSAGE_ALL][Wed Nov 27 09:28:04 CET 2019][net.sourceforge.jnlp.runtime.Boot] NETX Thread# 6bf2d08e, name main Proceeding with jnlp
[xxxxxxxxxx][ITW-JAVAWS][MESSAGE_ALL][Wed Nov 27 09:28:04 CET 2019][net.sourceforge.jnlp.runtime.Boot] NETX Thread# 6bf2d08e, name main JNLP file location: C:\Users\xxxxxxxxxx\AppData\Local\Microsoft\Windows\INetCache\IE\CUF5XDW5\my-application.DEV.jnlp
[xxxxxxxxxx][ITW-JAVAWS][MESSAGE_ALL][Wed Nov 27 09:28:04 CET 2019][net.adoptopenjdk.icedteaweb.xmlparser.MalformedXMLParser] NETX Thread# 6bf2d08e, name main Using MalformedXMLParser
[xxxxxxxxxx][ITW-JAVAWS][MESSAGE_ALL][Wed Nov 27 09:28:04 CET 2019][net.sourceforge.jnlp.Parser] NETX Thread# 6bf2d08e, name main Homepage: https://https://my-application.test
[xxxxxxxxxx][ITW-JAVAWS][MESSAGE_ALL][Wed Nov 27 09:28:04 CET 2019][net.sourceforge.jnlp.Parser] NETX Thread# 6bf2d08e, name main Description: null
[xxxxxxxxxx][ITW-JAVAWS][MESSAGE_DEBUG][Wed Nov 27 09:28:04 CET 2019][net.adoptopenjdk.icedteaweb.manifest.ManifestAttributesReader] NETX Thread# 6bf2d08e, name main Jars not ready to provide attribute Application-Name
[xxxxxxxxxx][ITW-JAVAWS][MESSAGE_ALL][Wed Nov 27 09:28:04 CET 2019][net.sourceforge.jnlp.JNLPFile] NETX Thread# 6bf2d08e, name main Acceptable title tag found, contains: My Application Installation
[xxxxxxxxxx][ITW-JAVAWS][MESSAGE_ALL][Wed Nov 27 09:28:04 CET 2019][net.sourceforge.jnlp.JNLPFile] NETX Thread# 6bf2d08e, name main Acceptable vendor tag found, contains: Daimler AG
[xxxxxxxxxx][ITW-JAVAWS][MESSAGE_ALL][Wed Nov 27 09:28:04 CET 2019][net.sourceforge.jnlp.Parser] NETX Thread# 6bf2d08e, name main good - your JRE - 1.8.0_212 - match requested JRE - 1.8*
[xxxxxxxxxx][ITW-JAVAWS][MESSAGE_DEBUG][Wed Nov 27 09:28:04 CET 2019][net.sourceforge.jnlp.JNLPFile] NETX Thread# 6bf2d08e, name main UNIQUEKEY=1574843284556-866784876-file:/C:/Users/xxxxxxxxxx/AppData/Local/Microsoft/Windows/INetCache/IE/CUF5XDW5/my-application.DEV.jnlp
[xxxxxxxxxx][ITW-JAVAWS][MESSAGE_ALL][Wed Nov 27 09:28:04 CET 2019][net.adoptopenjdk.icedteaweb.xmlparser.MalformedXMLParser] NETX Thread# 6bf2d08e, name main Using MalformedXMLParser
[xxxxxxxxxx][ITW-JAVAWS][MESSAGE_ALL][Wed Nov 27 09:28:04 CET 2019][net.sourceforge.jnlp.Parser] NETX Thread# 6bf2d08e, name main Homepage: https://https://my-application.test
[xxxxxxxxxx][ITW-JAVAWS][MESSAGE_ALL][Wed Nov 27 09:28:04 CET 2019][net.sourceforge.jnlp.Parser] NETX Thread# 6bf2d08e, name main Description: null
[xxxxxxxxxx][ITW-JAVAWS][MESSAGE_DEBUG][Wed Nov 27 09:28:04 CET 2019][net.adoptopenjdk.icedteaweb.manifest.ManifestAttributesReader] NETX Thread# 6bf2d08e, name main Jars not ready to provide attribute Application-Name
[xxxxxxxxxx][ITW-JAVAWS][MESSAGE_ALL][Wed Nov 27 09:28:04 CET 2019][net.sourceforge.jnlp.JNLPFile] NETX Thread# 6bf2d08e, name main Acceptable title tag found, contains: My Application Installation
[xxxxxxxxxx][ITW-JAVAWS][MESSAGE_ALL][Wed Nov 27 09:28:04 CET 2019][net.sourceforge.jnlp.JNLPFile] NETX Thread# 6bf2d08e, name main Acceptable vendor tag found, contains: Daimler AG
[xxxxxxxxxx][ITW-JAVAWS][MESSAGE_ALL][Wed Nov 27 09:28:04 CET 2019][net.sourceforge.jnlp.Parser] NETX Thread# 6bf2d08e, name main good - your JRE - 1.8.0_212 - match requested JRE - 1.8*
[xxxxxxxxxx][ITW-JAVAWS][MESSAGE_DEBUG][Wed Nov 27 09:28:04 CET 2019][net.sourceforge.jnlp.JNLPFile] NETX Thread# 6bf2d08e, name main UNIQUEKEY=1574843284565-1177390062-file:/C:/Users/xxxxxxxxxx/AppData/Local/Microsoft/Windows/INetCache/IE/CUF5XDW5/my-application.DEV.jnlp
[xxxxxxxxxx][ITW-JAVAWS][MESSAGE_DEBUG][Wed Nov 27 09:28:04 CET 2019][net.sourceforge.jnlp.runtime.JNLPRuntime] NETX Thread# 6bf2d08e, name main Acquired shared lock on C:\Users\xxxxxxxxxx\AppData\Local\Temp\xxxxxxxxxx\netx\locks\netx_running to indicate javaws is running
[xxxxxxxxxx][ITW-JAVAWS][MESSAGE_DEBUG][Wed Nov 27 09:28:04 CET 2019][net.adoptopenjdk.icedteaweb.manifest.ManifestAttributesReader] NETX Thread# 6bf2d08e, name main Jars not ready to provide attribute Application-Name
[xxxxxxxxxx][ITW-JAVAWS][MESSAGE_ALL][Wed Nov 27 09:28:04 CET 2019][net.sourceforge.jnlp.JNLPFile] NETX Thread# 6bf2d08e, name main Acceptable title tag found, contains: My Application Installation
[xxxxxxxxxx][ITW-JAVAWS][MESSAGE_DEBUG][Wed Nov 27 09:28:04 CET 2019][net.adoptopenjdk.icedteaweb.manifest.ManifestAttributesReader] NETX Thread# 6bf2d08e, name main Jars not ready to provide attribute Application-Name
[xxxxxxxxxx][ITW-JAVAWS][MESSAGE_ALL][Wed Nov 27 09:28:04 CET 2019][net.sourceforge.jnlp.JNLPFile] NETX Thread# 6bf2d08e, name main Acceptable title tag found, contains: My Application Installation
[xxxxxxxxxx][ITW-JAVAWS][MESSAGE_DEBUG][Wed Nov 27 09:28:04 CET 2019][net.sourceforge.jnlp.services.ServiceUtil] NETX Thread# 6876764b, name My Application Installation call privileged method: checkSingleInstanceRunning
[xxxxxxxxxx][ITW-JAVAWS][MESSAGE_DEBUG][Wed Nov 27 09:28:04 CET 2019][net.sourceforge.jnlp.services.ServiceUtil] NETX Thread# 6876764b, name My Application Installation arg: net.sourceforge.jnlp.JNLPFile@2d74f521
[xxxxxxxxxx][ITW-JAVAWS][MESSAGE_DEBUG][Wed Nov 27 09:28:04 CET 2019][net.sourceforge.jnlp.services.ServiceUtil] NETX Thread# 6876764b, name My Application Installation result: null
[xxxxxxxxxx][ITW-JAVAWS][MESSAGE_DEBUG][Wed Nov 27 09:28:04 CET 2019][com.openwebstart.launcher.OwsJvmLauncher] NETX Thread# 6876764b, name My Application Installation searching for JRE with version string '1.8*'
[xxxxxxxxxx][ITW-JAVAWS][MESSAGE_DEBUG][Wed Nov 27 09:28:04 CET 2019][com.openwebstart.jvm.JavaRuntimeSelector] NETX Thread# 6876764b, name My Application Installation Trying to find Java runtime. Requested version: '1.8*', vendor: '*', os: 'WIN64', server-url: 'https://download-openwebstart.com/jvms'
[xxxxxxxxxx][ITW-JAVAWS][MESSAGE_DEBUG][Wed Nov 27 09:28:04 CET 2019][com.openwebstart.jvm.LocalRuntimeManager] NETX Thread# 6876764b, name My Application Installation Trying to find local Java runtime. Requested version: '1.8*' Requested vendor: '*' requested os: 'WIN64' active: 'true'
[xxxxxxxxxx][ITW-JAVAWS][MESSAGE_DEBUG][Wed Nov 27 09:28:04 CET 2019][com.openwebstart.jvm.JavaRuntimeSelector] NETX Thread# 6876764b, name My Application Installation No local runtime found, will try to find remote runtime
[xxxxxxxxxx][ITW-JAVAWS][MESSAGE_DEBUG][Wed Nov 27 09:28:04 CET 2019][com.openwebstart.jvm.RemoteRuntimeManager] NETX Thread# 6876764b, name My Application Installation Trying to find remote Java runtime. Requested version: '1.8*' Requested vendor: '*' requested os: 'WIN64'
[xxxxxxxxxx][ITW-JAVAWS][MESSAGE_DEBUG][Wed Nov 27 09:28:04 CET 2019][com.openwebstart.jvm.RemoteRuntimeManager] NETX Thread# 6876764b, name My Application Installation Endpoint to request for Java runtimes: https://download-openwebstart.com/jvms
[xxxxxxxxxx][ITW-JAVAWS][MESSAGE_DEBUG][Wed Nov 27 09:28:04 CET 2019][net.sourceforge.jnlp.runtime.JNLPProxySelector] NETX Thread# 6876764b, name My Application Installation Selecting proxy for: https://download-openwebstart.com/jvms
[xxxxxxxxxx][ITW-JAVAWS][MESSAGE_ALL][Wed Nov 27 09:28:04 CET 2019][net.sourceforge.jnlp.browser.BrowserAwareProxySelector] NETX Thread# 6876764b, name My Application Installation Browser selected proxies: [DIRECT]
[xxxxxxxxxx][ITW-JAVAWS][MESSAGE_DEBUG][Wed Nov 27 09:28:04 CET 2019][net.sourceforge.jnlp.runtime.JNLPProxySelector] NETX Thread# 6876764b, name My Application Installation Selected proxies: [DIRECT]
[xxxxxxxxxx][ITW-JAVAWS][MESSAGE_DEBUG][Wed Nov 27 09:28:04 CET 2019][net.sourceforge.jnlp.runtime.JNLPProxySelector] NETX Thread# 6876764b, name My Application Installation Selecting proxy for: socket://download-openwebstart.com:443
[xxxxxxxxxx][ITW-JAVAWS][MESSAGE_ALL][Wed Nov 27 09:28:04 CET 2019][net.sourceforge.jnlp.browser.BrowserAwareProxySelector] NETX Thread# 6876764b, name My Application Installation Browser selected proxies: [DIRECT]
[xxxxxxxxxx][ITW-JAVAWS][MESSAGE_DEBUG][Wed Nov 27 09:28:04 CET 2019][net.sourceforge.jnlp.runtime.JNLPProxySelector] NETX Thread# 6876764b, name My Application Installation Selected proxies: [DIRECT]
[xxxxxxxxxx][ITW-JAVAWS][ERROR_ALL][Wed Nov 27 09:28:25 CET 2019][com.openwebstart.jvm.RemoteRuntimeManager] NETX Thread# 6876764b, name My Application Installation Error while trying to find a remote version: Connection timed out: connect
[xxxxxxxxxx][ITW-JAVAWS][MESSAGE_DEBUG][Wed Nov 27 09:28:25 CET 2019][com.openwebstart.jvm.LocalRuntimeManager] NETX Thread# 6876764b, name My Application Installation Trying to find local Java runtime. Requested version: '1.8*' Requested vendor: '*' requested os: 'WIN64' active: 'false'
[xxxxxxxxxx][ITW-JAVAWS][MESSAGE_ALL][Wed Nov 27 09:28:25 CET 2019][com.openwebstart.jvm.JavaRuntimeSelector] NETX Thread# 6876764b, name My Application Installation
Fehler beim Runterladen von 1.8* (Url=https://download-openwebstart.com/jvms)
java.lang.RuntimeException: Cannot provide or find runtime for version '1.8*', vendor '*' and operating system 'WIN64'
	at com.openwebstart.jvm.JavaRuntimeSelector.lambda$askForDeactivatedRuntime$5(JavaRuntimeSelector.java:86)
	at java.util.Optional.orElseThrow(Optional.java:290)
	at com.openwebstart.jvm.JavaRuntimeSelector.askForDeactivatedRuntime(JavaRuntimeSelector.java:86)
	at com.openwebstart.jvm.JavaRuntimeSelector.lambda$getRuntime$1(JavaRuntimeSelector.java:69)
	at java.util.Optional.orElseGet(Optional.java:267)
	at com.openwebstart.jvm.JavaRuntimeSelector.getRuntime(JavaRuntimeSelector.java:69)
	at com.openwebstart.jvm.JavaRuntimeSelector.getJavaRuntime(JavaRuntimeSelector.java:44)
	at com.openwebstart.launcher.OwsJvmLauncher.getJavaRuntime(OwsJvmLauncher.java:62)
	at com.openwebstart.launcher.OwsJvmLauncher.launchExternal(OwsJvmLauncher.java:52)
	at net.sourceforge.jnlp.Launcher.launchExternal(Launcher.java:320)
	at net.sourceforge.jnlp.Launcher.launchApplication(Launcher.java:405)
	at net.sourceforge.jnlp.Launcher.access$300(Launcher.java:69)
	at net.sourceforge.jnlp.Launcher$TgThread.run(Launcher.java:756)
[xxxxxxxxxx][ITW-JAVAWS][ERROR_ALL][Wed Nov 27 09:28:25 CET 2019][com.openwebstart.ui.ErrorDialog] NETX Thread# 5fc0eae7, name AWT-EventQueue-1
Error: Fehler beim Runterladen von 1.8* (Url=https://download-openwebstart.com/jvms)
java.lang.RuntimeException: Cannot provide or find runtime for version '1.8*', vendor '*' and operating system 'WIN64'
	at com.openwebstart.jvm.JavaRuntimeSelector.lambda$askForDeactivatedRuntime$5(JavaRuntimeSelector.java:86)
	at java.util.Optional.orElseThrow(Optional.java:290)
	at com.openwebstart.jvm.JavaRuntimeSelector.askForDeactivatedRuntime(JavaRuntimeSelector.java:86)
	at com.openwebstart.jvm.JavaRuntimeSelector.lambda$getRuntime$1(JavaRuntimeSelector.java:69)
	at java.util.Optional.orElseGet(Optional.java:267)
	at com.openwebstart.jvm.JavaRuntimeSelector.getRuntime(JavaRuntimeSelector.java:69)
	at com.openwebstart.jvm.JavaRuntimeSelector.getJavaRuntime(JavaRuntimeSelector.java:44)
	at com.openwebstart.launcher.OwsJvmLauncher.getJavaRuntime(OwsJvmLauncher.java:62)
	at com.openwebstart.launcher.OwsJvmLauncher.launchExternal(OwsJvmLauncher.java:52)
	at net.sourceforge.jnlp.Launcher.launchExternal(Launcher.java:320)
	at net.sourceforge.jnlp.Launcher.launchApplication(Launcher.java:405)
	at net.sourceforge.jnlp.Launcher.access$300(Launcher.java:69)
	at net.sourceforge.jnlp.Launcher$TgThread.run(Launcher.java:756)

```

**Comment by barthel** [comment]
no more logs available...

**Comment by aPollO2k** [comment]
The domain download-openwebstart.com is actually unused. But it appears in your log files. Do you compiled by yourself or used pre built installer? I have no trouble with 1.0.0 on windows 10

**Comment by barthel** [comment]
The domain ```https://download-openwebstart.com/jvms``` is the default value from which the Java VM is to be downloaded.

**Comment by hendrikebbers** [comment]
@barthel does this error still happens?

**Comment by barthel** [comment]
Yes.
![image002](https://user-images.githubusercontent.com/1170328/70411902-b5a02300-1a53-11ea-8a17-7dd3f9a8d3e9.jpg)

The JNLP file points to this URL and the OpenWebStart option to allow configuration of the JVM download in JNLP files is enabled.

**Comment by barthel** [comment]
Yes.
![image002](https://user-images.githubusercontent.com/1170328/70412206-7c1be780-1a54-11ea-8d62-99e69a562cd8.jpg)

The JNLP file points to this URL and the OpenWebStart option to allow configuration of the JVM download in the JNLP file is enabled.

**Comment by barthel** [comment]
If I follow this URL a valid JSON is shown with two JVM options:
```json
[...]
        {
            "version":"1.8.0_222",
            "vendor":"Zulu Community Edition",
            "os":"WIN64",
            "href":"https://download-openwebstart.com/Azul/zulu8.40.0.25-ca-fx-jdk8.0.222-win_x64.zip"
        },
[...]
        {
            "version":"1.8.0_222",
            "vendor":"AdoptOpenJDK",
            "os":"WIN64",
            "href":"https://download-openwebstart.com/AdoptOpenJDK/OpenJDK8U-jdk_x64_windows_hotspot_8u222b10.zip"
        },
```

**Comment by sclassen** [comment]
OK, I went through all the log and the code to find out what happens.
The cause seems to be the following:
```
Error while trying to find a remote version: Connection timed out: connect
```

So it appears to be network related.
In the log we also see
```
Endpoint to request for Java runtimes: https://download-openwebstart.com/jvms
Selecting proxy for: https://download-openwebstart.com/jvms
Browser selected proxies: [DIRECT]
```
This tells us that OpenWebStart is set to use the proxy settings from your local Firefox installation.
Since there is no error message it seams that OpenWebStart finds a Firefox profile and within that profile there is no Proxy defined `[DIRECT]`

No the question to @barthel: is this the correct proxy settings?
We did change the proxy detection with 1.1.0 so if the proxy settings are not correct my recommendations would be to
a) install version 1.1.0
b) go into the settings and validate the proxy configuration

On the other hand: If the proxy settings are already correct then the next step would be to detect on a network level which packages are sent back and forth. Only this allows to further search for the cause of the connection timeout.
